js/gateio.js

```diff
@@ -1823,25 +1824,41 @@ module.exports = class gateio extends Exchange {
 
     fetchBalanceHelper (entry) {
         const account = this.account ();
-        account['used'] = this.safeString2 (entry, 'locked', 'position_margin');
+        let used = this.safeString (entry, 'freeze');
+        if (used === undefined) {
+            used = this.safeString2 (entry, 'locked', 'position_margin');
+        }
+        account['used'] = used;
         account['free'] = this.safeString (entry, 'available');
         return account;
     }
 
     async fetchBalance (params = {}) {
-        // :param params.type: spot, margin, crossMargin, swap or future
+        // :param params.type: spot, margin, cross, swap or future
         // :param params.settle: Settle currency (usdt or btc) for perpetual swap and future
         await this.loadMarkets ();
         let type = undefined;
+        let method = undefined;
         [ type, params ] = this.handleMarketTypeAndParams ('fetchBalance', undefined, params);
+        if (type === 'margin') {
+            const defaultMarginType = this.safeString2 (this.options, 'defaultMarginType', 'marginType', 'default');
```

Comment:
@kroitor Actually does not make a difference because we're comparing against 'cross' but I'll change it

Reply:
It actually does make a difference, since we have just two unified margin types: `cross` and `isolated`. However, I think, this exchange defaults to cross-margin. What is the default margin mode used with Gate.io when you login to their website and pick the most popular BTC perpetual contract?

Reply:
@kroitor opening the BTC/USDT margined pair I get this by default
![image](https://user-images.githubusercontent.com/43336371/161510233-1aea42fc-7147-42a3-9ba0-c84d0398c760.png)